### PR TITLE
Fixes redirect_uri for python oauth

### PR DESCRIPTION
**Pull request by skansal-plaid**
Tested locally to make the required changes.

### CODE REVIEW

File: python/server.py [hunk 1]
```diff
@@ -30,21 +30,14 @@
 PLAID_COUNTRY_CODES = os.getenv('PLAID_COUNTRY_CODES', 'US').split(',')
 
 
-def empty_to_none(field):
-  value = os.getenv(field)
-  if value is None or len(value) == 0:
```
Comment (skansal-plaid) [inline python/server.py:35]: if len(value) is zero, then developer shouldn't have specified an empty redirect uri in the first place

Comment (davidzhanghp) [reply python/server.py:35]: do you think that if i run a command with PLAID_REDIRECT_URI= <command> that it should be invalid?

Comment (skansal-plaid) [reply python/server.py:35]: yep, specifying an empty redirect uri for oauth is not valid

**Comment by skansal-plaid** [comment]
Closing in favor of https://github.com/plaid/quickstart/pull/140

**Pull request closed without merging**